src/pages/q/[[...all]].tsx

```diff
@@ -85,6 +85,11 @@ const SearchIndex: any = ({
     initialTopicData,
   )
 
+  // HIDE_DUPLICATES: the `loadInstructor` call here is only pulling in
+  // instructor metadata. So, if we have duplicate instructors living in
+  // egghead-rails and we are only pulling in one of them (based on slug), then
+  // as long as we are relying on the metadata for both duplicate instructors
+  // being the same, this won't be an issue.
```

Comment:
this isn't an exercise of deduplication IMO but more of a "if PSEUDO use REAL"

```
if (i.slug === 'fake-kcd') i.slug = 'real-kcd'
```

on the client, instructor meta is only for display and we don't want to have duplicate instructors we want to point the fake instructor to the real one's meta.